includes/DataObjects/Request.php

```diff
@@ -263,12 +271,12 @@ public function hasComments()
 			return true;
 		}
 
-		$commentsQuery = $this->dbObject->prepare("SELECT COUNT(*) as num FROM comment where request = :id;");
-		$commentsQuery->bindValue(":id", $this->id);
+		if (is_null(self::$commentsQuery)) self::$commentsQuery = $this->dbObject->prepare("SELECT COUNT(*) as num FROM comment where request = :id;");
```

Comment:
Aside from the line length making this hard to review, it's also hard to see what's happening. Bytes are cheap - please use braces.

I'm also wondering if pushing this up to DataObject in a clean way would help alleviate the inevitable explosion of fields here - I'm thinking of a method where we can pass in a key, and it uses the current class name to look up in a cache. Again, I'd like other input on this before we go ahead, as it'll be a pretty big change. @FastLizard4 ?